docs/linting/README.md

```diff
@@ -16,10 +19,10 @@ npm install --save-dev eslint typescript @typescript-eslint/parser @typescript-e
 
 ## Configuration
 
-Next, create a `.eslintrc.js` config file in the root of your project, and populate it with the following:
+Next, create a `.eslintrc.cjs` config file in the root of your project, and populate it with the following:
```

Comment:
Confirmed that ESLint supports `.cjs`: https://eslint.org/docs/user-guide/configuring/configuration-files. I think it's reasonable to start telling people to use `.cjs` since it is already supported and is future-proof for the CJS contents within.